packages/core/data-transfer/lib/providers/local-file-destination-provider/utils.ts

```diff
@@ -0,0 +1,76 @@
+import { Writable } from 'stream';
```

Comment:
Could we merge this file with the test utils we have at a higher level so we can re-use it easier?

Reply:
These are not utils for testing, these are utils to abstract them from the local file destination provider and unit test them separately, do you want me to move the utils file into a higher level anyways?

Reply:
Ahh, I was confused. It makes more sense to keep these here where you put them, nothing else should need to use these.